client/src/api/notifications.js

```diff
@@ -8,7 +8,7 @@ export default {
     return {
       level: 0,
       heading: "The project has been deleted.",
-      body: `Alas, it has passed into the endless night.`,
+      body: `Alas, the project has passed into the endless night. However, it may take a minute or two for all traces to disappear from the frontend.`,
       expiration: defaultExpiration
     };
   },
```

Comment:
At some point, I think it might be nice to replace this list of notifications that are handled via the redux store. They're easy to miss, and now that we're doing most of this stuff with hooks, it's nice to be able to customize the data passed into the messages in the relevant component.

Reply:
I agree. Then we can also localize these messages.